cli/src/wizards/wallet.rs

```diff
@@ -1,13 +1,16 @@
 use crate::cli::KaspaCli;
 use crate::imports::*;
 use crate::result::Result;
-use kaspa_wallet_core::runtime::{PrvKeyDataCreateArgs, WalletCreateArgs};
-use kaspa_wallet_core::storage::{make_filename, AccessContextT, AccountKind, Hint};
+use kaspa_bip32::{Language, Mnemonic, WordCount};
+use kaspa_wallet_core::storage::{make_filename, Hint};
 
 pub(crate) async fn create(ctx: &Arc<KaspaCli>, name: Option<&str>, import_with_mnemonic: bool) -> Result<()> {
     let term = ctx.term();
     let wallet = ctx.wallet();
 
+    // TODO @aspect
```

Comment:
The original behavior of the mnemonic was to treat 12 words as legacy and 24 words as core (`'111111`).  Since it was voted to support and default to 12 words as core, the processing pathways have changed and this needs to be reviewed, and possibly the wizard needs to be made to ask if the user is restoring legacy vs core.  Unfortunately, I don't have time right now to focus on this so I am deferring this.